libraries/libfc/src/crypto/bls_public_key.cpp

```diff
@@ -19,40 +27,37 @@ namespace fc::crypto::blslib {
       constexpr bool raw = false;  // non-montgomery
       std::optional<bls12_381::g1> g1 = bls12_381::g1::fromAffineBytesLE(bytes, check, raw);
       FC_ASSERT(g1);

```

Comment:
Updated `bls_public_key`. Unfortunately we need `bls_public_key::operator=` because things like `finalizer_policy` require it. Let me know if you think the `const` is worth it.

Reply:
Actually, I forgot that fc reflection does not work with `const`, so will remove the `const`.